### DISCUSSION TITLE
enable elrs RX with pwm converter to be considered as native PWM RX

### DISCUSSION THREAD

**Discussion by psychotik2k3** [ExpressLRS/ExpressLRS#2031] [Ideas]
Hi i will use ELRS on my numerous gliders and to do that i have to use at least 6 channel for pure gliders (elevator, rudder (or vstab 1 and vstab2), aileron1, aileron2 flap1 and flap 2 and yes i need all of them for quadroflaps, snapflap or crow brakes) and 7 for motor gliders.
unfortunately because of the 2 postion only channel 5 i cannot use 6 pwm converters.
I read on the wiki that native PWML receivers can change their mapping of the channels to get rid of the channel 5 (but for now all pwm native receivers get only 5 channels).
Is there a way in the wifi configuration to force the RX to be considered as a native PWM (and setup number of output channels) ?
This way i can use the new 100hz or 333hz full mode to have my servos used correctly.

thanks

**Comment by miles2912** [comment]
The problem is that we don't have any PWM RX's that use more than 5 servos.   When they make an 8 channel + batt we should be good.  Now we are stuck with 5.  BTW - mapping out channel 5 is easy.  So we get a full range on all 5 channels.

**Reply by psychotik2k3** [reply]
That's exactly the problem for now the only solution is to go with crsf to pwm converters. But 6 channels converters are  quite useless because of channel 5 if we cannot change its source

**Comment by Gerold68** [comment]
I am also a wing pilot and so fore, I like this MATEKSYS converters most.
http://www.mateksys.com/?portfolio=crsf-pwm#tab-id-4
10 ch. support plus some more nice wing features.
Anyway, I expect native 10Ch. RX in the near future.

**Comment by psychotik2k3** [comment]
I have 2 of them and 4 of the 6 channel conversion.
Unfortunately the 6 channels won't helps me for the pure gliders, only for
the ones with no flaps. That's why is find this a waste not to be able to
have the mapping if it's not a native pwm rx.
